index.js

```diff
@@ -131,9 +131,11 @@ class HtmlWebpackPlugin {
       ...global,
       HTML_WEBPACK_PLUGIN: true,
       require: require,
+      __non_webpack_require__: require,
```

Comment:
I guess we don't need this as webpack will convert `__non_webpack_require__` to `require` - or did you find any scenario where this might be useful?

Reply:
You mentioned it in the issue so I just blindly added it😬. You're right though, webpack already creates the mapping to `require`